### ISSUE TITLE
API_Msoro.ino: "start" commands for 10 or more gait cycles

### ISSUE THREAD

**Issue by achang46** [arunniddish/API#1]
[API_Msoro.ino] Line 86: gnum = T_d1.charAt(1); 

Line appears to assume single-digit gait execution duration (i.e. # of gait cycles). 

For example, API.m will send following commands over serial: "start" followed by 'A7'. This commands Arduino to execute gait A for 7 gait cycles. Current implementation (API_Msoro.ino, line 86) doesn't appear to support gait durations of 10 or more gait cycles (e.g. 'A11' appears as if it will execute gait A for 1 gait cycle, not 11 gait cycles).

**Comment by arunniddish** [comment]
Yes, I noted this issue when i was coding. What I had in my mind - for suppose if you had to give 'A12', then the command needs to be passed as summation of 2 or more sequence. ('A8'+'A4') ('A9'+'A3') etc.

**Comment by achang46** [comment]
Having to command 'A8' followed by 'A4' (instead of just 'A12') is kind of unwieldy; believe it can lead to potential system issues down the road as well (more complexity in any application commanding MSoRo, timing considerations, ...).

Is there a reason the Arduino can't just process up to 2-digit (or even 3-digit, if you ever think we/you will ever run 100+ gait cycles) gait execution durations?

Believe if we re-structure how commands are processed (i.e. [Git issue 5](https://github.com/arunniddish/API/issues/5)), this issue can be resolved along the way.

**Comment by achang46** [comment]
[Meeting, Alex-Arun, 4/15/22] 
- Will support up to 2 or 3-digit gait execution durations (as a parameter for the "start" command)
- Will implement when re-structuring command processing (i.e. [Git issue 5](https://github.com/arunniddish/API/issues/5))

Began an interface description document that we can collaboratively finalize: 
https://docs.google.com/document/d/1NhcKvrKJ3Sy_6hmfkjQHOzd4s-_9TxEv1JlvnkAAXpk/edit#

**Comment by achang46** [comment]
Arun implemented capability -> action on Alex to verify via Matlab MSoRo interface
